docs-src/docs/annotation-integration/review.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Review
+---
+
+A review is a decision that determines whether an [annotation](../#annotation) is of sufficient quality or if it needs 
+improvement. In order to facilitate efficient improvements, the person or system that made the decision can provide 
+[feedback](#feedback) data that details potential errors or mistakes. Please refer to our [user documentation](https://docs.kognic.com/AHwD-introduction) 
+regarding feedback to gain insight on how these concepts are used in the platform, note that these describe the full
+set of possibilities and workflows for dealing with reviews and feedback - the possibilities enabled by this integration
+are currently somewhat limited, as detailed below. 
+
+## Posting a review
+
+The current integration capabilities only allow posting reviews for delivered annotations. The annotations are 
+identified using their open label UUID that can be found in the metadata (and in the file name) of [downloaded 
+annotations](../kognic-io/annotations).
+
+When posting a review, the API expects feedback data as well as a boolean `accepted` that indicates whether the 
+quality is perceived as sufficient or not. The API also expects an enum `workflow` that details how the improvement 
+should happen. For a rejected review and the workflow 'correct', a single correction task is triggered, with no 
+follow-up Review task. The completed Correction task becomes the new delivered Annotation. 
+
+:::caution
+This integration API is currently limited to rejecting delivered annotations, the possible values of these input 
+parameters are thus limited to `accepted=false` and `workflow="correct"`. They are included in the interface at this
+stage in order to allow future expansions on the available use cases, workflows and actions.
```

Comment:
In terms of modelling, would `action` be a suitable interface for this?

That way you could bundle the `accept=false & workflow="correct"` into a `RejectWithCorrection` enum or Object (depending on further settings is required or not). That way the combination is explicit in an `action` object rather than combination of parameters

Reply:
The reason being that for the Qualcomm use case, I guess the `accept` boolean doesn't really apply since the action is really `review`?

Reply:
We want to make the accept-param optional in the future, thereby allowing just adding feedback for later consumption. I think that interface feels more intuitive than having to come up with an action name for every possible POST/PATCH of these review resources. And then the workflow parameter would be optional as well.

Reply:
> I imagined that as a different endpoint, since they don't want to post the actual review but add data to help during the review. Or maybe they want to "start" or "trigger" a review?
> 
> But then I don't have a good design in my head for that approach either.

You might be right. I guess this is me thinking about is as the "integration phase decision endpoint" discussed loosely some time ago. So a bit of hopium on my end and interpretation of the possible "workflows".